packages/app/src/cli/models/app/identifiers.ts

```diff
@@ -45,19 +47,22 @@ interface UpdateAppIdentifiersOptions {
  * @returns An copy of the app with the environment updated to reflect the updated identifiers.
  */
 export async function updateAppIdentifiers(
-  {app, identifiers, command}: UpdateAppIdentifiersOptions,
+  {app, identifiers, command, developerPlatformClient}: UpdateAppIdentifiersOptions,
   systemEnvironment = process.env,
 ): Promise<AppInterface> {
-  await Promise.all(
-    app.realExtensions.map(async (extension) => {
-      if (extension.isUuidManaged() && (await fileExists(extension.configurationPath))) {
-        const tomlContents = await readFile(extension.configurationPath)
-        const extensionConfig = decodeToml(tomlContents)
-        if ('uid' in extensionConfig) return
-        await writeFile(extension.configurationPath, `uid = "${extension.uid}"\n${tomlContents}`)
-      }
-    }),
-  )
+  if (developerPlatformClient.supportsAtomicDeployments) {
+    await Promise.all(
+      app.realExtensions.map(async (extension) => {
```

Comment:
Does this handle gracefully a case where multiple extensions are defined in one file? Seems like that would fail somehow.

Reply:
I don't think it would work. But neither the previous solution, with a single UID for several extensions that would be deployed separately, no?

We have never had yet a template with multiple extensions, so I'm not even sure if the production code will work. We have been trying to support it for months, but it's still unused, does it make sense to keep pushing? Or should we wait and try to make it work once we are sure that it will happen? cc: @isaacroldan

Reply:
Right. The thing is that the current approach is generating a single UUID per template. But let me try to improve it.

Reply:
I've finally made it by looking for the right handle of each extension, which is unique.